docroot/sites/default/settings.php

```diff
@@ -199,5 +199,16 @@
   $deploy_service->run($GLOBALS['request'], $app_root, $site_path);
 }
 
+if (PHP_SAPI === 'cli') {
+  // This is running from drush so set the webhost.
+  $webhost = $webhost_on_cli;
+}
+else {
+  //This is running from a web request.
+  $webhost = "{$_SERVER['REQUEST_SCHEME']}://{$_SERVER['HTTP_HOST']}";
+
+}
+ $settings['file_public_base_url'] = "{$webhost}/sites/default/files";
```

Comment:
Remove leading space.
```suggestion
$settings['file_public_base_url'] = "{$webhost}/sites/default/files";
```